src/components/LoginForm.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { FormEvent, useContext, useState } from 'react';
+import Cookies from 'js-cookie';
+import {
+  Box,
+  Button,
+  Form,
+  FormField,
+  Heading,
+  HelperErrorText,
+  TextInput,
+} from '@nypl/design-system-react-components';
+import { TokenContext, TokenContextValues } from '../context/tokenContext';
+
+const backgroundStyles = {
+  background: 'section.blogs.primary',
+  margin: 'auto',
+  marginTop: 'xxl',
+  paddingBottom: 'xl',
+  paddingTop: 'l',
+  width: '400px',
+};
+const formFieldStyles = { margin: 'auto', width: '75%' };
+const errorStyles = {
+  color: 'ui.error.primary',
+  paddingTop: 'xs',
+  textAlign: 'center',
+};
+
+const LoginForm = () => {
+  const [username, setUsername] = useState<string>('');
+  const [password, setPassword] = useState<string>('');
+  const [showLoginError, setShowLoginError] = useState<boolean>(false);
+
+  const { setAccessToken } = useContext(TokenContext) as TokenContextValues;
+
+  const login = (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    const body = new FormData();
+    body.append('username', username);
+    body.append('password', password);
+
+    fetch(process.env.QA_LOGIN as RequestInfo | URL, {
```

Comment:
Hmm I don't see the `.env` file in these changes. Maybe it was added but not committed to git? Since it's not a secret, we can go ahead and commit this file.